app/assets/javascripts/discourse/app/controllers/user-activity-bookmarks.js

```diff
@@ -15,7 +15,7 @@ export default Controller.extend({
 
   application: controller(),
   user: controller(),
-  loading: false,
+  loading: true,
```

Comment:
@martin-brennan Sorry I just realised it is safer for this to be `false` because in the route we may return a cache. When that happens, the ajax request isn't called so we will never reach the `finally` hook.